src/main/java/millom/sandbox/service/WeatherService.java

```diff
@@ -0,0 +1,26 @@
+package millom.sandbox.service;
+
+import millom.sandbox.mapper.NasaMapper;
+import millom.sandbox.pojos.Weather;
+
+public class WeatherService {
+  MyClientService myClientService = new MyClientService();
+  NasaMapper nasaMapper = new NasaMapper();
+
+  public WeatherService(MyClientService myClientService, NasaMapper nasaMapper) {
+    this.myClientService = myClientService;
+    this.nasaMapper = nasaMapper;
+  }
+  public WeatherService(){
```

Comment:
What is this ctor for?

Reply:
this ctor is used in the method **getMarsWeather() of NasaResource** class to make use of the method getWeatherMapping(feed,feedType,version,category) in WeatherService class.